### ISSUE TITLE
Use aria-describedby to associate errors with the inputs they describe

### ISSUE THREAD

**Issue by klemay** [hypothesis/product-backlog#782]
Part of: https://github.com/hypothesis/product-backlog/issues/753 

Currently,  error messages for login inputs are not read out to assistive tech users because they are not associated with their inputs (screen reader output shown below).

<img width="288" alt="screen shot 2018-08-30 at 5 11 25 pm" src="https://user-images.githubusercontent.com/12087705/44879461-bf937d80-ac77-11e8-9e35-d9ec53642602.png">

Used `aria-describedby` for all error messages.

As far as I know, in the client, we show error messages for:
- username/email
- password

**Comment by ajpeddakotla** [comment]
| User Engagement | Building Community | Level of Effort |
|---|---|---|
| 2 | 1 | 3 |

**Comment by ajpeddakotla** [comment]
Closed in favor of #1085
